src/parser.js

```diff
@@ -17,9 +17,23 @@
 function Parser (filename, grammar) {
     this.filename = filename;
     this.grammar = grammar;
+    this.p_flags = 0;
     return this;
 }
 
+// all possible parser flags
```

Comment:
Those flags are required when we add parsing of future statements and passing flags to compile function.